documentation/src/main/asciidoc/topics/marshalling.adoc

```diff
@@ -93,225 +66,84 @@ include::config_examples/serialization_marshaller.xml[]
 include::code_examples/SerializationMarshaller.java[]
 ----
 
-=== Troubleshooting
-Sometimes it might happen that the {brandname} marshalling layer, and in particular JBoss Marshalling, might have issues marshalling/unmarshalling some user object. In {brandname} 4.0, marshalling exceptions will contain further information on the objects that were being marshalled. Example:
+=== JBoss Marshalling
+JBoss Marshalling is a Serialization based marshalling library which was the default marshaller in past versions of {brandname}.
+It's possible to still utilise this library, however it's necessary to first add the `infinipsan-jboss-marshalling` dependency
+to the classpath. No additional configuration is required, as if this module is on the class path then the user marshaller defaults
+to using JBoss Marshalling.
```

Comment:
I'd call out this is deprecated at a higher level.

/[IMPORTANT]/
/====
JBoss Marshalling is deprecated and planned for removal in a future version of {brandname}.
====/